packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -592,7 +610,7 @@
     <parameter>
       <name>aggregation: coloring algorithm</name>
       <type>string</type>
-      <default>serial</default>
+      <default>Zoltan2</default>
```

Comment:
Could this lead to issues? Serial is supposedly deterministic.

Reply:
Yeah, but Zoltan2 works in parallel.